open-machine-learning-jupyter-book/_toc.yml

```diff
@@ -59,7 +59,7 @@ parts:
 #     - file: ml-fundamentals/regression/managing-data
 #     - file: ml-fundamentals/regression/linear-and-polynomial-regression
 #     - file: ml-fundamentals/regression/logistic-regression
-#   - file: ml-fundamentals/web-app
+  - file: ml-fundamentals/Build a Web App to use a Mchine Learning model

```

Comment:
Please follow the naming convention for the files. This should be `build-a-web-app-to-use-a-machine-learning-model`. Also, please make sure there is no typo. Mchine -> Machine